sphinxcontrib/drawio.py

```diff
@@ -288,20 +268,34 @@ def on_build_finished(app: Sphinx, exc: Exception) -> None:
         os.waitid(os.P_PID, app.builder.config.xvfb_pid, os.WEXITED)
 
 
+FALLBACK_EXPORT_FORMAT = "png"
+DEFAULT_BUILDER_EXPORT_FORMAT = dict(html="svg", latex="pdf", rinoh="pdf")
```

Comment:
I'd prefer if this was a normal dictionary expression, i.e.:
```suggestion
DEFAULT_BUILDER_EXPORT_FORMAT = {
	"html": "svg",
	"latex": "pdf",
	"rinoh": "pdf",
}
```

Reply:
Done.